homeassistant/components/intent/__init__.py

```diff
@@ -85,6 +90,24 @@ async def async_call_service(self, intent_obj: intent.Intent, state: State) -> N
             )
             return
 
+        if state.domain == LOCK_DOMAIN:
+            # on = lock
+            # off = unlock
+            await self._run_then_background(
+                hass.async_create_task(
+                    hass.services.async_call(
+                        LOCK_DOMAIN,
+                        SERVICE_LOCK
+                        if self.service == SERVICE_TURN_ON
```

Comment:
Ternary operator expressions that span more than one line are really hard to read. I suggest saving a local variable above first and use that here.